pkg/data/sqlite_test.go

```diff
@@ -57,7 +57,7 @@ func TestAddEpisodeHistory(t *testing.T) {
 	e := EpisodeAction{
 		Podcast:   "http://podcast.com/rss.xml",
 		Episode:   "episode 232",
-		Device:    device,
+		Device:    device.Id,
```

Comment:
I don't think we should use `device.Id` here because 

in:
```
err = data.AddEpisodeActionHistory(username, e)
```

We are actually expecting the `device.Name` instead of `Id`

https://github.com/oxtyped/gpodder2go/blob/main/pkg/data/sqlite.go#L120-L123

This is causing the test to be unable to retrieve the device because of https://github.com/oxtyped/gpodder2go/blob/main/pkg/data/sqlite.go#L174

Reply:
from the specs, it seemed to me this should have been the `ID`. but ok :)